### PR TITLE
PHP7 compatibility

### PR DESCRIPTION
**Pull request by Jan-E**

### CODE REVIEW

File: uploadprogress.c [hunk 1]
```diff
@@ -301,7 +305,7 @@ PHP_FUNCTION(uploadprogress_get_info)
 {
     char * id;
     int id_lg;
```
Comment (laruence) [inline uploadprogress.c:310]: this should be size_t id_lg

**Comment by Jan-E** [comment]
Fixed with https://github.com/Jan-E/pecl-php-uploadprogress/commit/95d8a0fd4554e10c215d3ab301e901bd8f99c5d9

**Comment by Jan-E** [comment]
@ramsey Here is the PHP7 pull request

**Comment by ramsey** [comment]
Thanks, @Jan-E. I'm going to see if I can get the tests hooked up to run automatically in Travis CI before merging in. Might be a couple of days.

**Comment by ramsey** [comment]
Well, I thought there were tests. I'll need to add some. :-)

**Comment by ramsey** [comment]
In the meantime, I'll go ahead and merge this and then work on getting tests written. Thanks!

**Comment by Jan-E** [comment]
There weren't any tests. I just did an upload of a 900MB file to Apache on my laptop to see if it worked as expected. It did, even without memory loss: https://github.com/Jan-E/uploadprogress/commit/50d4a16be3fcf4ef8909f4a4785d1416c4de81fe#commitcomment-10643800

**Pull request merged**